artiq/frontend/artiq_master.py

```diff
@@ -154,6 +156,11 @@ def ccb_issue(service, *args, **kwargs):
         bind, args.port_logging))
     atexit_register_coroutine(server_logging.stop)
 
+    core_host, core_port = device_db.get('core')['arguments']['host'], 1383
+    core_proxy = AsyncProxyServer((core_host, core_port))
```

Comment:
Other APIs in ARTIQ/sipyco use two arguments for host and port, and not a tuple.